### DISCUSSION TITLE
Tag Node (beta)

### DISCUSSION THREAD

**Discussion by zachowj** [zachowj/node-red-contrib-home-assistant-websocket#348] [General]
Tag Node (beta)

Sends an output when Home Assistant receives a tag scanned event for a configured tag id.

- trigger from multiple tags
- only trigger if from a defined device

**Comment by ervede** [comment]
Awesome node!!
Within the properties (when editing the node) there are options to configure:
- Tag (tag_name)
- Devices (device_id)

My suggestion is to put user_id also to this list 
With the expectation that when you block a user in HA it will also stop this user to enable the tag.

**Reply by zachowj** [reply]
How do you block a user in HA? 

I could see adding a user as a possible condition to the node. I think it would need to follow the same conventions as tag_name and device_id do now. Where the user that scanned the tag would need to be on the user list.

What you looking for could be done but using a subflow or if it's a one-off API node to a switch node.

**Comment by jimmhead** [comment]
Great node, makes scanning tags so much easier.

My suggestion would be to have the option to work with any tag rather than defining the list of tags (similar to listening to all tag_scanned events, but with the tag name included)

I've got a series of nfc cards to play music, the tag name has everything included in it to play the correct album or playlist through node red so I don't need to set anything up for each new tag, however I currently have to add each tag to the list in the node to get it to work

**Comment by kocmocgim** [comment]
I don't see tag's name
![2021-09-28_11-40-27](https://user-images.githubusercontent.com/24375347/135036128-97da9c85-4f7e-48cc-80eb-6d582f224d71.png)

**Reply by zachowj** [reply]
Does it fill in if you click on the `Update Tag List`?

**Reply by kocmocgim** [reply]
If i set a specific name for tag in home assistant

**Reply by zachowj** [reply]
I was under the impression that you couldn't create a tag in the UI without giving it a name. 

I'll change it so if the name doesn't exist it will display the tag id.

**Reply by kocmocgim** [reply]
Thank you 👍

**Comment by webbson** [comment]
I would like it to be possible to allow any tags to be scanned, so that there is a simple way of getting the name of the card currently scanned. 

Then I could replace my own function with this. Today I have to listen to tag_scanned event, then call the tag/list websocket and then filter out the correct tag from there to be able to show the name of the tag scanned.

**Reply by zachowj** [reply]
That was added in v0.35.0 and the tag name should exists in the `eventData`
